### PR TITLE
Fix typo in a password reset token defendpoint docstring

### PR DESCRIPTION
**Pull request by nemanjaglumac**

### CODE REVIEW

File: src/metabase/api/session.clj [hunk 1]
```diff
@@ -296,7 +296,7 @@
       (api/throw-invalid-param-exception :password (tru "Invalid reset token"))))
 
 (api/defendpoint GET "/password_reset_token_valid"
-  "Check is a password reset token is valid and isn't expired."
+  "Check is a password reset token valid and isn't expired."
```
Comment (paoliniluis) [inline src/metabase/api/session.clj:299]: Shouldn't it be "Check if a password reset token is valid and isn't expired"

**Review by dragonsahead — CHANGES_REQUESTED** [review]
Left 1 comment

File: src/metabase/api/session.clj (hunk 1 above)
Comment (nemanjaglumac) [reply src/metabase/api/session.clj:299]: I think both work.
But I do like your suggestion better.

**Review by dragonsahead — APPROVED** [review]
LGTM!

**Pull request merged**